### DISCUSSION TITLE
Deprecated SNS integration

### DISCUSSION THREAD

**Discussion by dani-mp** [vercel/community#559] [Help] [answered]
Hey!

We were wondering how to subscribe to our SNS topic on each preview of our app when we found [this integration](https://vercel.com/integrations/sns), which is supposed to do exactly what we need.

It's deprecated, though, and we can't use it anymore. Is there any alternative?

**Comment by samcx** [comment, marked as answer]
Unfortunately, I am not aware of any alternative. If you wish to create another one again since the old one was deprecated, feel free to do so! → https://vercel.com/docs/integrations/create-integration

**Comment by dani-mp** [comment]
@samsisle we're sponsoring the developer who created the original integration to update it to the new APIs so it can be published.

**Reply by samcx** [reply]
Awesome!
